test/unittests/evm_fixture.hpp

```diff
@@ -35,46 +35,55 @@
 class evm : public testing::Test
 {
 protected:
+    /// The VM handle.
     evmc::vm& vm;
-    evmc_revision rev = EVMC_BYZANTIUM;  // Byzantium by default. TODO: Add alias evmc::revision.
-    evmc_message msg = {};               // TODO: Add evmc::message with default constructor.
-    evmc::result result{{}};  // TODO: Add default constructor to evmc::result, update code here.
+
+    /// The EVM revision for unit tests execution. Byzantium by default.
+    /// TODO: Add alias evmc::revision.
+    evmc_revision rev = EVMC_BYZANTIUM;
+
+    /// The message to be executed by an unit test (with execute() method).
+    /// TODO: Add evmc::message with default constructor.
```

Comment:
msg isn’t invoking the default evmc_message ctor?

Reply:
It's a C struct.